web/packages/teleport/src/Apps/AddApp/Automatically.tsx

```diff
@@ -105,8 +97,25 @@ export function Automatically(props: Props) {
                 placeholder="https://localhost:4000"
                 onKeyPress={e => handleEnterPress(e, validator)}
                 onChange={e => setUri(e.target.value)}
+                disabled={attempt.status === 'processing'}
               />
             </Flex>
+            <Box mt={-3} mb={3}>
```

Comment:
Why -3?

Reply:
the component above this, the [FieldInput](https://github.com/gravitational/teleport/blob/f63a099ca797ac6c82b86190504c8e2fbe795ef4/web/packages/shared/components/FieldInput/FieldInput.tsx#L101) adds `mb={3}` that I needed to remove to make things looked spaced correctly